source/sass/wagtail.scss

```diff
@@ -69,6 +69,14 @@
       max-height: 800px;
     }
   }
+
+  .blog-body {
```

Comment:
We can do this in SCSS here or modify every non-heading HTML tags in `blog-body` so they are all assigned with the `body-large` class.

e.g., 

```html
<p class="body-large"></p>
<ul class="body-large"></ul>
```

Reply:
Here sounds nice

Reply:
What we have here should be sufficient enough.
```scss
.blog-body {
    .block-paragraph { // classname for Wagtail 'paragraph' component
      p,
      ul,
      ol,
      a {
        @extend .body-large;
      }
    }
}

```